client-go/cmd/apps.go

```diff
@@ -0,0 +1,245 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/deis/deis/client-go/controller/client"
+	"github.com/deis/deis/client-go/controller/models/apps"
+	"github.com/deis/deis/client-go/controller/models/config"
+)
+
+// AppCreate creates an app with the application
+func AppCreate(id string, buildpack string, remote string, noRemote bool) error {
+	c, err := client.Load()
+
+	fmt.Print("Creating Application... ")
+	quit := progress()
+	app, err := apps.Create(*c, id)
+
+	quit <- true
+	<-quit
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("done, created %s\n", app.ID)
+
+	if buildpack != "" {
+		configValues := map[string]string{
+			"BUILDPACK_URL": buildpack,
+		}
+		err = config.Set(*c, app.ID, configValues)
```

Comment:
I think Deis go code prefers the one-line form for error checks if you don't need to refer to the return values later. (We should have this in our coding standards doc if there is consensus on this.) Doesn't have to be changed; just sayin'.
```go
if err = config.Set(*c, app.ID, configValues); err != nil {
        return err
}
```